web/src/components/vis/Heatmap.vue

```diff
@@ -664,54 +673,107 @@ export default {
 };
 </script>
 
-<template lang="pug">
-.grid(v-resize:throttle="onResize")
-  svg.column(ref="column", v-show="columnConfig.dendrogram",
-      :width="matrixWidth",
-      :height="height * DENDROGRAM_RATIO", xmlns="http://www.w3.org/2000/svg",
-      :data-update="reactiveColumnUpdate")
-    g.edges(:transform="`translate(0,${padding})`")
-    g.nodes(:transform="`translate(0,${padding})`")
-  svg.row(ref="row", v-show="rowConfig.dendrogram",
-      :width="width * DENDROGRAM_RATIO",
-      :height="matrixHeight", xmlns="http://www.w3.org/2000/svg",
-      :data-update="reactiveRowUpdate")
-    g.edges(:transform="`translate(${padding},0)`")
-    g.nodes(:transform="`translate(${padding},0)`")
-  canvas.matrix(ref="matrix", :data-update="reactiveMatrixUpdate",
-      @mousemove="canvasMouseMove($event)", @mouseleave="canvasMouseLeave()")
-  .collabel(ref="collabel",
-      :style="{fontSize: fontSize+'px', width: this.matrixWidth+'px', height: LABEL_WIDTH+'px'}",
-      :data-update="reactiveColumnLabelUpdate")
-  .rowlabel(ref="rowlabel",
-      :style="{fontSize: fontSize+'px', width: LABEL_WIDTH+'px', height: this.matrixHeight+'px'}",
-      :data-update="reactiveRowLabelUpdate")
-  .legend-wrapper(v-show="columnConfig.dendrogram && rowConfig.dendrogram")
-    .legend(:data-from="legendDomain[0]", :data-to="legendDomain[1]",
-        :style="{background: legendGradient}")
+<template>
+  <div
+    v-resize:throttle="onResize"
+    :class="`grid${layout === 'full' ? ' grid--large' : ''}`"
```

Comment:
Don't use string interpolation for dynamic classes, there's a better way.

```suggestion
    :class="{
      'grid': true,
      'grid--large': layout === 'full',
    }"
```